visual_testing_wdio/wdio.chrome.conf.js

```diff
@@ -0,0 +1,27 @@
+const baseConfig = require('./wdio.conf.js').config
+const args = process.env.IS_HEADLESS_MODE ? headlessModeArgs : []
```

Comment:
No need for headless mode as stated before. Also remove "goog:chromeOptions" since it will be empty